mobs/fox.lua

```diff
@@ -1,3 +1,11 @@
+local followf = {}
+
+table.insert(followf, "mobs:meat_raw")
+
+if minetest.get_modpath("mobs_animal") then
+	table.insert(followf, "mobs_animal:muttom_raw")
+end
+
```

Comment:
i dont know if #8 must be closed in flavor of this or i must clean this?

Reply:
> The mod checks for `drops` are a good idea _if_ you are making the dependencies optional. If you make the dependencies mandatory, you do not need to add mod checks... because the dependency in `mod.conf` guarantess their presence already.

solved .. with 3a2176f  now changed as requested. i will close the #8